13.0/Dockerfile

```diff
@@ -0,0 +1,78 @@
+FROM debian:stretch
+MAINTAINER Camptocamp
+
+# create the working directory and a place to set the logs (if wanted)
+RUN mkdir -p /odoo /var/log/odoo
+
+COPY ./base_requirements.txt /odoo
+COPY ./install /install
+
+# Moved because there was a bug while installing `odoo-autodiscover`. There is
+# an accent in the contributor name
+ENV LANG=C.UTF-8 \
+    LC_ALL=C.UTF-8
+
+# build and dev packages
+ENV BUILD_PACKAGE \
+    build-essential \
+    gcc \
+    python3.5-dev \
+    libevent-dev \
+    libfreetype6-dev \
+    libxml2-dev \
+    libxslt1-dev \
+    libsasl2-dev \
+    libldap2-dev \
+    libssl-dev \
+    libjpeg-dev \
+    libpng-dev \
+    zlib1g-dev \
+    git
+
+# Install some deps, lessc and less-plugin-clean-css, and wkhtmltopdf
+RUN set -x; \
+        sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ stretch-pgdg main" > /etc/apt/sources.list.d/pgdg.list' \
+        && /install/package_odoo_11.0_12.0.sh \
```

Comment:
I'm not fond of the nomenclature, but it includes the name of all versions where it is used.
Feel free to ignore my nitpicking comment.